.github/workflows/code-pushup.yml

```diff
@@ -21,6 +21,7 @@ jobs:
       CP_API_KEY: ${{ secrets.CP_API_KEY }}
       CP_ORGANIZATION: code-pushup
       CP_PROJECT: cli
+      CUSTOM_CHROME_PATH: C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe
```

Comment:
The `code-pushup` job only runs on `ubuntu-latest`, so I guess setting this environment variable is a mistake.

Reply:
removed